sheet_to_triples/rdf.py

```diff
@@ -40,26 +38,33 @@ def _n3(uri, namespace_manager):
     return rdflib.URIRef(uri).n3(namespace_manager)
 
 
-def from_qname(qname, namespaces=rdflib.namespace):
-    if qname.startswith('vm:'):
-        return VM[qname[3:]]
-    if qname.startswith('http:'):
+def from_qname(qname, resolver):
+    prefix, tail = qname.split(':', 1)
+    namespace = resolver(prefix)
+    if namespace:
+        return rdflib.URIRef(namespace + tail)
+    # TODO: Is this special casing for http: strictly needed now?
```

Comment:
The tests unfortunately rely on it quite a bit as otherwise the implicit assumption is that everything that passes through this function has a prefix, which means you need to teach the all of the tests that touch this part of the code (which is most of them) about prefixing behaviour so that you can test the output properly. Prefixing with `http` grants a straight qname in -> qname out (albeit wrapped inside `URIRef`) relationship, which makes testing much easier and less brittle to changes elsewhere.